bin/pull-latest-release-asset.sh

```diff
@@ -3,8 +3,20 @@
 repoName=$1
 assetName=$2
 
-latestAssetId=$(curl https://api.github.com/repos/hmcts/${repoName}/releases/latest \
-  | docker run --rm --interactive stedolan/jq ".assets[] | select(.name==\"${assetName}\") | .id")
+retries=0
+while [ -z "$latestRelease" ]
+do
+  latestRelease=$(curl https://api.github.com/repos/hmcts/${repoName}/releases/latest)
+  retries=$((retries+1))
```

Comment:
Can we add `echo` call here outputting retry number? Might be useful to know how many attempts it took to pull the asset.